src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -2014,6 +2014,11 @@ ert_configure_cu(struct sched_cmd *cmd, int cu_idx)
 
 	sk = (struct ert_start_kernel_cmd *)cmd->packet;
 
+	if (opcode(cmd) == ERT_EXEC_WRITE)
+		type = PAIRS;
+	else
+		type = CONSECUTIVE;
+
```

Comment:
one nit, you can remove the "= CONSECUTIVE“ at line 2010.

Reply:
Done.